mdn/developer-portal
# 1416 Make custom streamfield optional in two places

This changeset makes sure that the followign streamfield items are no longer mandatory, as we may not always have content to put in them:

* ContentPage -> Sidebar
* Events -> Body

Both of these should have been made optional when the fields were added (by me). This resolves the oversight.

## How to test

Code is up on dev. 
- Please confirm that you can save (or even just preview, because that's enough) an Events page (ie the listing page) with no items at all in its `body` streamfield
- Please confirm that you can save (or even just preview, because that's enough) a ContentPage page (eg About) with no items at all in its `sidebar` streamfied
